.circleci/config.yml

```diff
@@ -18,14 +18,17 @@ jobs:
           at: .
       - rn/yarn_install
       - run:
-          name: Run ESLint
-          command: yarn lint
+          name: Run Tests
+          command: yarn test
```

Comment:
Not sure if it can really be helpful, but maybe we can add a "Run Typecheck" step with `yarn tsc --noEmit` so that we can stop early if there are TS errors. 
What do you think?

Reply:
I think it's an excellent idea to enforce typechecking rules. To be done on next commit